### DISCUSSION TITLE
價格波動&時間的關係

### DISCUSSION THREAD

**Discussion by Will150143** [ChiuAlfredo/options_day_data#25] [研究]
# 研究題目
同價性選擇權價格報酬率在各時段是否存在差異

## 研究方法
使用結算日當天各時段的報酬率進行單因子變異數分析(One-way ANOVA)，驗證各時段的報酬率是否存在差異 α=0.05
時間切分為8.45-9.、9.-9.10、9.10-9.30、9.30-10.，之後半小時切一直到13.30
- H0 : μt1 = μt2 = μt3= ... = μtn 各組平均數皆相等
- H1 : 至少有兩組之間的平均數不相等
## 使用資料
價內、價平、價外買和賣權分報酬率，分報酬率計算方式為：Ln(當分鐘成交價)-Ln(前一分鐘成交價)後將同價性的分報酬率取平均

## 預期結果
- 選擇權報酬率各時段皆相同，無法拒絕H0
- 選擇權報酬率各時段間存在差異，拒絕H0

## 補充資料
- 單因子變異數分析(One-way ANOVA)
  - 介紹
單因子變異數分析(ANOVA)用於比較多組之間的平均數差異
  - 公式
![image](https://github.com/ChiuAlfredo/options_day_data/assets/133984319/f45450a7-a6f1-4e0c-94b3-188077f78580)
  - 事後檢定
若檢定結果為顯著，會進行事後比較確認差異是在哪一組之間
常見方法有很多種例如Tukey HSD、Bonferroni、Scheffe以及LSD，其他方法和說明可參閱以下網站：
https://www.ibm.com/docs/zh-tw/spss-statistics/beta?topic=anova-one-way-post-hoc-tests

**Comment by ChiuAlfredo** [comment]
1. 要使用什麼方式做比較價格波動的標準差，完全沒說明
2. (P2-P1)/P1不明確

**Reply by ChiuAlfredo** [reply]
1.如公式所示，可以說一下你指的說明是什麼意思嗎
2.不明確是什麼意思，該分鐘價格-前一分鐘價格/前一分鐘價格，是這樣嗎？

**Reply by ChiuAlfredo** [reply]
> 1.如公式所示，可以說一下你指的說明是什麼意思嗎 2.不明確是什麼意思，該分鐘價格-前一分鐘價格/前一分鐘價格，是這樣嗎？

1. 算出來波動率，然後要做什麼比較，沒有寫
2. 這個價格是每一分鐘的波動率還是什麼

**Comment by TsuYiii** [comment]
1.選擇權價格取值範圍：結算日當天的價平（0.99<S/K<1.01)買、賣權 時間單位為1分鐘。應為K/S嗎？
